ops/cli.py

```diff
@@ -275,6 +295,32 @@ def print_deployment_success(config: WflInstanceConfig) -> None:
         run_liquibase_migration,
         stop_cloud_sql_proxy,
         print_deployment_success
+    ],
+    "ops-deploy": [
+        infer_missing_arguments_pre_validate,
+        validate_cloud_sql_name,
+        validate_cluster_name,
+        infer_missing_arguments,
+        check_git_tag,
+        configure_kubectl,
+        configure_helm,
+        render_values_file,
+        print_config,
+        exit_if_dry_run,
+        configure_cloud_sql_proxy,
+        helm_deploy_wfl,
+        run_liquibase_migration,
+        stop_cloud_sql_proxy,
+        print_deployment_success
+    ],
+    "hornet-pre-deploy": [
```

Comment:
Similar to above, how about naming this to something more descriptive like "tag-and-push-images"?